src/patternfly/components/Table/table-tree-view.scss

```diff
@@ -9,7 +9,8 @@ $pf-c-tree-view--MaxDepth: 10;
   --pf-c-table__tree-view-main--MarginLeft: calc(var(--pf-c-table--cell--PaddingLeft) * -1);
 
   // check
-  --pf-c-table__tree-view-main--c-table__check--MarginRight: var(--pf-global--spacer--sm);
```

Comment:
Removing this will break core users since they don't have the new `<label>`

<img width="319" alt="Screen Shot 2022-02-01 at 11 04 50 AM" src="https://user-images.githubusercontent.com/35148959/152015562-67bd9a8c-12b0-4f5c-a07e-8cbbc9b3a927.png">

Reply:
@mcoker Whoops, updated.